### DISCUSSION TITLE
Tropikal meyve kurutma sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#24] [Zeka Sorusu] [answered]
Tropikal bir meyvenin % 99’u sudur. Bu meyveden 500 gram kurutulmak üzere güneşte bırakılıyor. Bir süre sonra içindeki su oranı % 98 olduğu saptanıyor.

Soru: Kaç gram meyve kalmıştır?

**Comment by ArvenAA** [comment, marked as answer]
Eğer 500 gramlık bir meyvenin 99%'u su ise 5 gram meyvenin özü, 495 gram ise meyvenin suyudur. Su oranının 98%'e düşmesi ile meyvenin özünün oranı 2%'ye çıkar. Bu durumda meyvenin özünün ağırlığı değişmediği için 2/100 = 5 gram olur, meyvenin tamamı 250 gram, meyvenin suyu ise 245 gram olur. Yani sorunun cevabı 250.
